### PR TITLE
Update yii2-apidoc dependency to the latest version

### PR DESCRIPTION
**Pull request by arogachev**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
     "homepage": "https://www.yiiframework.com/",
     "type": "project",
     "license": "BSD-3-Clause",
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```
Comment (samdark) [inline composer.json:8]: We should add `prefer-stable` as well.

File: composer.json [hunk 2]
```diff
@@ -70,13 +70,16 @@
         "process-timeout": 1800,
         "fxp-asset": {
            "enabled": false
-        },
-        "platform": {"php": "7.0"}
```
Comment (samdark) [inline composer.json:74]: That should be updated to what is actually on the server.

Comment (arogachev) [reply composer.json:74]: OK. Let's wait for https://github.com/yiisoft-contrib/yiiframework.com/issues/960 to be resolved.

**Review by Arhell — APPROVED** [review]

**Comment by arogachev** [comment]
Closing in favor of #962.

**Pull request closed without merging**